prow/cmd/tackle/ingress_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	extensions "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	discoveryFake "k8s.io/client-go/discovery/fake"
+	k8sFake "k8s.io/client-go/kubernetes/fake"
+	"testing"
+	"time"
+)
+
+func TestNetworkingIngressNotExists(t *testing.T) {
```

Comment:
I'd be inclined to combine these two tests into a single table driven test (like the rest of our repo tends to do)

https://github.com/kubernetes/community/blob/master/contributors/guide/coding-conventions.md#testing-conventions